.github/workflows/integration-tests.yaml

```diff
@@ -12,11 +12,47 @@ jobs:
         os: [ubuntu-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - name: Checkout the current branch of mizu-client repo here.
+        uses: actions/checkout@v3
+
+
+      - name: Set up Docker Buildx
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
What is the thought process behind only testing Docker and linux and only testing non-docker deployment on Windows?

Reply:
currently, i am not able to find a way to build windows container image with docker.
So, just went with docker image build only on linux for now.
Let me know if there is a way to build windows container images with docker or any other builder.

Reply:
My understanding is that the os field here is for the host machine, not the docker image. 

I think we want to use the integration tests to verify that Mizu runs on both windows and linux, and that the docker file works. And thinking about it, the way you have it set up here accomplishes all of those goals.